do-not-commit/return-in-init.py

```diff
@@ -0,0 +1,37 @@
+class A:
+    def __init__(a, b, c):
+        # ruleid:return-in-init
+        return A(a, b, c)
+
+
+class A:
+    def __init__(a, b, c):
+        # ruleid:return-in-init
+        return
+
+
+class B:
+    def __init__(a, b, c):
+        # todoruleid:yield-in-init
+        yield
```

Comment:
`yield` should never appear inside a class __init__ function. This will cause a runtime error. ([.return-in-init.yield-in-init](https://semgrep.dev/r/.return-in-init.yield-in-init))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=.return-in-init.yield-in-init) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>